src/operator/random/sample_op.h

```diff
@@ -249,6 +254,30 @@ struct SampleGenNegBinomialParam : public dmlc::Parameter<SampleGenNegBinomialPa
   }
 };
 
+struct SampleRandIntParam : public dmlc::Parameter<SampleRandIntParam>,
+  RandIntParam, SampleOpParam {
+  DMLC_DECLARE_PARAMETER(SampleRandIntParam) {
+    DMLC_DECLARE_FIELD(low).set_default(0)
+    .describe("Lower bound of the distribution.");
+    DMLC_DECLARE_FIELD(high).set_default(1)
```

Comment:
These low and high defaults may not be appropriate for this operator. What do you expect them to get out of these?

Reply:
removed defaults.